### DISCUSSION TITLE
Add standardized code formatting

### DISCUSSION THREAD

**Discussion by holmbuar** [Mikubill/sd-webui-controlnet#1485] [Ideas]
In my fork of this repo, I prefer to apply code formatting. In particular I like to break up long code lines, to avoid horizontal scrolling and increase readability.

My preferred Python formatter is Black, one of many available. 

https://pypi.org/project/black/

Once I apply formatting though, and update my fork with latest changes from origin, the result is a whole lot of merge conflicts. 

Therefore I kindly suggest that the repo contributors choose and apply a code formatter. Perhaps only in smaller/ shorter files as a start.

**Comment by huchenlei** [comment]
We can probably add a git presubmit hook to format code.
